controllers/snapshot/snapshot_controller.go

```diff
@@ -156,5 +159,8 @@ func setupControllerWithManager(manager ctrl.Manager, reconciler *Reconciler) er
 		For(&applicationapiv1alpha1.Snapshot{}).
 		WithEventFilter(predicate.Or(
 			gitops.IntegrationSnapshotChangePredicate())).
+		WithOptions(controller.Options{
+			RateLimiter: workqueue.NewItemExponentialFailureRateLimiter(2*time.Second, 1*time.Hour),
```

Comment:
Why this change? Was default rate limit 2-100sec not enough?

Reply:
I wanted to find the place to set the default rate limit but I didn't find it. Can you help to paste the link. I want to read it. Thanks!

Reply:
https://github.com/kubernetes/client-go/blob/master/util/workqueue/default_rate_limiters.go#L39 This one

I had a typo in my previous msg, 1000sec (~17min what we see in logs)

Reply:
Thanks very much. Let me remove it.

Reply:
Removed.